homeassistant/components/emoncms/const.py

```diff
@@ -7,6 +7,7 @@
 CONF_MESSAGE = "message"
 CONF_SUCCESS = "success"
 DOMAIN = "emoncms"
+EMONCMS_UUID_DOC_URL = "https://docs.openenergymonitor.org/emoncms/update.html#upgrading-to-a-version-producing-a-unique-identifier"
```

Comment:
I think this could be split to fit in less than 88 chars
```suggestion
EMONCMS_UUID_DOC_URL = (
    "https://docs.openenergymonitor.org/emoncms/update.html"
    "#upgrading-to-a-version-producing-a-unique-identifier"
)
```